Gulpfile.js

```diff
@@ -156,6 +157,9 @@ gulp.task('html-hint', () => {
 
 // Test task
 gulp.task('test', ['check-handlebars', 'standard', 'html-hint'], () => {
+  env.set({
+    PORT: 8001
```

Comment:
Could we change this to `PORT: process.env.PORT + 1` instead. We're still assuming that port is free, but at least it relates to whatever we have in the config?

Reply:
Done - I have managed to read in the WASTE_PERMITS_APP_PORT from the .env file and then increment it by one when running the gulp tests.